WebApplication1/Models/Movie.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WebApplication1.Models
+{
+    public class Movie
+    {
+        public int ID { get; set; }
```

Comment:
The naming convension in .NET C# is a pascal case, where only the first letter is capital and if it consists of multiple words, the first letter of each word is capital, so here it should be "Id"